scripts/plotting/create_bar_plots.py

```diff
@@ -10,7 +10,7 @@
     create_dataframes, get_df_for_entry, pd_create_equal_selector
 
 plt.style.use('ggplot')
-pd.options.mode.chained_assignment = None  # default='warn'
+pd.options.mode.chained_assignment = None  # type: ignore
```

Comment:
an alternative here could be to use `pd.set_option('chained_assignment',None)` as shown [here](https://stackoverflow.com/questions/21463589/pandas-chained-assignments)